### PR TITLE
Add CLI and new Lambda UI instructions

### PR DESCRIPTION
**Pull request by JakeMKelly**
*Issue #, if available: NA*

*Description of changes: Add CLI template and new Lambda-build UI instructions, added .ask/config*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

@SleepyDeveloper

### CODE REVIEW

File: instructions/1-build-and-customize.md [hunk 1]
```diff
@@ -155,52 +155,36 @@ walkthrough for setting it up</a>.
...
 
-12.  **Configure your trigger.** Click in the dashed box, and select Alexa Skills 
-Kit from the list.  If you don't see Alexa Skills Kit in the list, jump back to 
-step #3 on this page.   
-    <img src="https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/general/2-6-configure-your-trigger._TTH_.png" />  
-    Once you have selected Alexa Skills Kit, click the **Submit** button.    
+Once you have selected Alexa Skills Kit, scroll down and click the **Add** button. Then click the **Save** button in the top right. You should see a green success message at the top of your screen. Now, click the box that has the Lambda icon followed by the name of your function (HighLowGame if you used our suggestion) and scroll down to the field called "Function code".
 
-13. **After you create the function, the ARN value appears in the top right 
+7.  **Copy and paste the [provided code](../lambda/custom/index.js) into the Lambda function code box.**  We have provided the code for this skill on [GitHub](../lambda/custom/index.js).  Delete the contents of the code box, and paste the contents of the new code.
```
Comment (SleepyDeveloper) [inline instructions/1-build-and-customize.md]: Numbering has come out of sync. It shows 7 but should be 11

**Comment by SleepyDeveloper** [comment]
Looks good other than the numbering.

File: instructions/1-build-and-customize.md (hunk 1 above)
Comment (JakeMKelly) [reply instructions/1-build-and-customize.md]: Ah, md preview corrects those- fix coming for this and the following number.

**Comment by JakeMKelly** [comment]
Should be good. 🔢

**Comment by SleepyDeveloper** [comment]
Thanks, found another issue. HighLowGame needs to be pet match.

**Comment by JakeMKelly** [comment]
Got it.

**Comment by SleepyDeveloper** [comment]
Thanks! LGTM

**Pull request merged**